agr_literature_service/api/crud/reference_crud.py

```diff
@@ -702,3 +702,27 @@ def download_tracker_table(db: Session, mod_abbreviation: str, order_by: str, fi
     # return FileResponse(path=tmp_file_with_path, filename=tmp_file, media_type='application/plain')
     return FileResponse(path=tmp_file_with_path, filename=tmp_file, media_type='application/plain',
                         background=BackgroundTask(cleanup, tmp_file_with_path))
+
+
+def get_bib_info(db, curie, mod_abbreviation: str, return_format: str = 'txt'):
+    bib_info = BibInfo()
+    reference: ReferenceModel = get_reference(db, curie, load_authors=True)
+    author: AuthorModel
+    for author in reference.author:
```

Comment:
Is this ordered by author."order" ?  Also, with the short citation stuff that Ian's working on, some authors don't have a last_name / first_initial, just a name.

Reply:
Thanks @azurebrd! I'll sort the authors by order. I'm not sure about author names. What format is used for the 'name' field?